src/applications/personalization/profile/components/ProfileMobileSubNav.jsx

```diff
@@ -202,7 +202,7 @@ const ProfileMobileSubNav = ({ isLOA3, isInMVI, routes }) => {
             <>
               <div className="menu-header vads-u-display--flex">
                 <strong className="vads-u-flex--auto">
-                  <h1 className={menuButtonClasses}>Your profile</h1> menu
+                  <h1 className={menuButtonClasses}>Profile</h1> menu
```

Comment:
@tressaellen I assume we also want the "Your profile menu" to be changed to "Profile menu", but since it was not in the ticket, could you please confirm? Thanks!

![image](https://user-images.githubusercontent.com/14869324/110413528-40067680-804b-11eb-9bd7-65ad62b5f1a1.png)